.github/workflows/pull_request.yml

```diff
@@ -140,15 +140,26 @@ jobs:
       runs-on: ubuntu-latest
       steps:
         - uses: actions/checkout@v2
+        - name: Set Tag and SHA
+          run: |
+            echo "::set-env name=TAG::${{ github.event.pull_request.head.ref }}"
+            echo "::set-env name=SHA::${{ github.event.pull_request.head.sha }}"
         - name: Write app version
-          run: printf "${{ github.event.pull_request.head.sha }}" > .application-version
-        - name: Tag
-          run: echo "::set-env name=TAG::${{ github.event.pull_request.head.ref }}"
+          run: |
+            echo "Writing SHA $SHA to .application_version"
+            printf $SHA > .application-version
         - name: Build
-          run: docker build -t onsdigital/eq-questionnaire-runner:$TAG .
-        - name: Push
+          run: >
+            docker build -t onsdigital/eq-questionnaire-runner:$TAG -t eu.gcr.io/census-eq-ci/eq-questionnaire-runner:$TAG
```

Comment:
Would be good to remove the census specific GCR reference like you did in launcher.